src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx

```diff
@@ -372,4 +372,7 @@
   <data name="PX1056IsActiveForGraphMethod" xml:space="preserve">
     <value>PXGraphCreationInIsActiveForGraphMethod</value>
   </data>
+  <data name="PX1062" xml:space="preserve">
+    <value>StaticPropertyOrFieldInGraph</value>
```

Comment:
@EAndrosova  please check the diagnostic short name

Reply:
approved